src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java

```diff
@@ -496,8 +497,8 @@ public JDBCTaskMetadata getCompletedMetadata() {
         try (var connection = getConnection(); var statement = getTaskInfoQuery(previous, taskRange, connection)) {
             var rs = statement.executeQuery();
             if (rs.next()) {
-                var max = tableInfo.hasIncColumn() ? rs.getLong("max") : 0;
-                var min = tableInfo.hasIncColumn() ? rs.getLong("min") : 0;
+                var max = tableInfo.hasIncColumn() ? rs.getLong(queryDialect.toUpperCaseIfRequired("max")) : 0;
+                var min = tableInfo.hasIncColumn() ? rs.getLong(queryDialect.toUpperCaseIfRequired("min")) : 0;
```

Comment:
This can just uppercase everywhere always

Reply:
Will also resolve needing to pass the query dialect to the row reader

Reply:
done